### PR TITLE
[WIP]:Changes in docs for 1.5

### PR DESCRIPTION
**Pull request by humblearner**

### CODE REVIEW

File: docs/source/start.rst [hunk 1]
```diff
@@ -40,8 +40,8 @@ The installation deploys the CLI along with the |st2| services. CLI can be used
 service remotely. All |st2| operations are also available via REST API, Python, and JavaScript
 bindings. Check the :doc:`CLI and Python Client </reference/cli>` reference for details.
 
-From v0.8, |st2| ships with WebUI. With the default installation, you can access it at
-http://hostname. If SSL is configured, the WebUI will redirect to https://hostname.
+|st2| ships with WebUI. You can access it at http://hostname. If SSL is configured, the WebUI
+will redirect to https://hostname.
```
Comment (lakshmi-kannan) [inline docs/source/start.rst]: this redirect probably doesn't work. I'd just point them to https://

**Comment by lakshmi-kannan** [comment]
lgtm except for that one comment.

File: docs/source/start.rst (hunk 1 above)
Comment (humblearner) [inline docs/source/start.rst]: ok. will change it.

**Comment by humblearner** [comment]
I am not done yet!! :)

**Comment by Kami** [comment]
I will go ahead and just merge it.

Please feel free to open a new PR with new improvements and fixes :)

**Comment by humblearner** [comment]
Whaaaat !! you closed it instead of merging it @Kami :)

**Comment by armab** [comment]
It's pretty hot in Europe :sunny: )

**Pull request closed without merging**